### PR TITLE
Add My Profile: Ayush-Kanduri.json

### PR DESCRIPTION
**Pull request by Ayush-Kanduri**
This PR will add the Ayush-Kanduri.json file to the '/public/data' Folder.

## Changes proposed

Ayush-Kanduri.json file added to the directory '/public/data'.

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots

Preview

## Note to reviewers

Kindly Merge my PR. I would be glad.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1892"></a>

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🎉

File: public/data/Ayush-Kanduri.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      "icon": "linkedin"
+    },
+    {
+      "name": "Subscribe to my channel on YouTube",
+      "url": "https://www.youtube.com/user/MrAyushChamp",
+      "icon": "youtube"
+    },
+    {
+      "name": "Checkout my Portfolio",
+      "url": "https://ayushkanduri.dev",
```
Comment (kumarsonsoff3) [inline public/data/Ayush-Kanduri.json:29]: There is a broken link, please update it with the correct one! 

image

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: public/data/Ayush-Kanduri.json (hunk 1 above)
Comment (Ayush-Kanduri) [reply public/data/Ayush-Kanduri.json]: Hi Priyanshu. Resolved the error. Kindly do the needful.

**Comment by Ayush-Kanduri** [comment]
Rectified the error. Kindly merge the PR.

**Review by EmmaDawsonDev — APPROVED** [review]

**Pull request merged**